Media Kit (#110)
readthedocs/ethicalads.io: content/pages/press.md

Title: Media Kit
slug: press

> description: Are you part of the tech press and you want to write about advertising that doesn't track you? This is the right place.

Reviewer: ```suggestion
description: Are you a journalist that wants to write about advertising that doesn't track you? This is the right place.
```

template: ea/press

> Want to speak with someone at EthicalAds? Please get in touch [below](#inbound-form).

Reviewer: We should include a basic style guide here around how to refer to us. Eg.

```suggestion
Thanks for your interest in writing about EthicalAds & Read the Docs.

### Quick Style Guide

* Our brand name is `EthicalAds` when referencing us. You can use `ethical advertising` to talk about the category or concept. 
* Our parent company is `Read the Docs`, which can be abbreviated at `RTD`.

### Contact us

Want to speak with someone at EthicalAds? Please get in touch [below](#inbound-form).
```
